app/assets/stylesheets/credits.scss

```
@@ -247,6 +247,15 @@
     border-radius: 3px;
     text-align: center;
 
+    @media screen and ( min-width:949px ) {
+      margin: 67px auto;
```

I think we can easily drop top margin - I can see no benefit on small breakpoint from this gap between header and that section...
```suggestion
      margin: 0 auto $su-6;
```

`$su-6` [stands for 1.5rem](https://github.com/thepracticaldev/dev.to/blob/master/app/assets/stylesheets/config/_import.scss#L86).